contentlayer.config.ts

```diff
@@ -2,48 +2,48 @@ import {
   defineDocumentType,
   defineNestedType,
   makeSource,
-} from 'contentlayer/source-files';
+} from "contentlayer/source-files";
```

Comment:
Did we change the formatter settings?

Reply:
Yea very messy. Ill fix this on my own machine and change it back